### PR TITLE
Added win/osx/linux cross-compilation via xgo

### PR DESCRIPTION
**Pull request by ignasbernotas**
cd $GOPATH/src/github.com/mysterium/node/
docker build -t mysteriumnetwork/mysterium-client ./bin/util_docker/builder
./bin/package_all 0.1.1

It creates osx/linux/windows binaries, and packages debian files, but fails at creating docker images.

### CODE REVIEW

File: bin/util_docker/builder/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,8 @@ RUN apt-get update \
 
 # Install Debber
 RUN go get github.com/debber/debber-v0.3/cmd/debber
+RUN go get github.com/oschwald/geoip2-golang
 
-WORKDIR /go/src/github.com/mysterium/node
+WORKDIR /
```
Comment (Waldz) [inline bin/util_docker/builder/Dockerfile]: This path was mounted in `builder_run`

File: bin/package_all [hunk 2]
```diff
@@ -9,40 +9,69 @@
...
     && printf "\n\n"
 
-for ARCH in amd64 i386 armhf
-do
-    bin/builder_run "bin/server_build; bin/server_package_debian $VERSION $ARCH" \
-      && mv build/server/mysterium_server $DIR_BUILD/mysterium_server_linux_$ARCH \
-      && printf "\n\n"
-done
+# build client
+docker run --rm -v $(pwd):$DOCKER_LOCAL_DIR $DOCKER_IMAGE -c "
```
Comment (Waldz) [inline bin/package_all]: You can assume that script is already inside docker, and call this script with:
`bin/builder_run bin/package_all`.
Script ecpects environment to be ready and calls `xgo` directly without proxying to docker

File: bin/package_all [hunk 3]
```diff
@@ -9,40 +9,69 @@
...
+    xgo --targets='$PLATFORMS'            \
+        --ldflags='$(get_linker_ldflags)' \
+        --out='mysterium_client'          \
```
Comment (Waldz) [inline bin/package_all]: --out=$DIR_BUILD

File: bin/package_all (hunk 2 above)
Comment (ignasbernotas) [reply bin/package_all]: This is so confusing, we're separating files into different responsibilities, but not reusing them out of package_all script.. I would prefer single larger file than many one liners

**Pull request closed without merging**